app/server.rb

```diff
@@ -0,0 +1,88 @@
+
+require 'sinatra/base'
+require 'data_mapper'
+require 'rack-flash'
+require 'sinatra/partial'
+
+require './app/models/peep' 
+require './app/models/user'
+
+require_relative 'data_mapper_setup'
+require_relative 'helpers/application'
+
+class Chitter < Sinatra::Base
+
+include Helpers
+
```

Comment:
Need to register partials as you are using a modular sinatra application (see https://github.com/yb66/Sinatra-Partial):

``` register Sinatra::Partial ```